### ISSUE TITLE
Cursor is hidden by a keyboard

### ISSUE THREAD

**Issue by Aga-C** [SimpleMobileTools/Simple-Notes#446]
Hi,

I've found a slightly irritating issue while writing longer notes. When I want to put a cursor somewhere at the bottom of the screen, the app doesn't scroll its view, so the cursor is hidden by the keyboard. On the video below, you can see how it looks (OnePlus 9 Pro with AnySoft Keyboard):

https://user-images.githubusercontent.com/85929121/133920747-dcab16be-01af-4dd3-9065-b420c60770b7.mp4

I've also checked on:
- OnePlus 8 Pro with SwiftKey and Gboard, Android 11
- OnePlus 2 with AnySoft, Android 6.0.1

Issue was present on both of them. However, the issue is not always reproducible. Sometimes it just works, but more often it's just not working, as I've shown on the video.
 
On the emulator (with standard keyboard and Android 11) issue only occurs when an empty line is selected. 

On every device and emulator, I've used the latest version of the app (on devices the version from F-Droid, on emulator the latest from Git).

**Comment by AkshaiBaruah** [comment]
Hello! I would like to work on this Issue. Please assign it to me .

**Comment by tibbi** [comment]
should work fine already

**Comment by moneytoo** [comment]
I'm still experiencing the issue in version 6.15.2. Trying it on Samsung Galaxy S21 FE (Android 13).

https://user-images.githubusercontent.com/1071643/215283929-abb05ee5-c9b9-44b9-b487-4989f7de8147.mp4

**Comment by Aga-C** [comment]
I'm still having this issue on my Android 13.

**Comment by adityab98** [comment]
I don't think this is specific to simple notes. I recently got very annoyed with this bug and tried various other apps. Notally on Fdroid also has this issue. To my surprise, even Google Keep had this issue! I wonder if there is some android or google-based API that is the source of this bug.

There is an app called Editor by Bill Farmer on Fdroid that doesn't have this issue. Could help pinpoint what exactly is causing it. Maybe this app (Editor) uses a different API or something?
